test/helpers/sharedBootstrap.js

```diff
@@ -62,6 +62,8 @@ module.exports = {
       'centres',
       'subjects',
       'movement',
+      'events',
```

Comment:
swap these round and you'll be able to do the easy linking fixtures as per previous comment